### DISCUSSION TITLE
Installed applications only recognized as Admin via CMD/Powershell

### DISCUSSION THREAD

**Discussion by EpicLPer** [Devolutions/UniGetUI#1632] [General]
Heya,

For the past months I've been wondering why I wasn't able to access some programs via CMD/Powershell after installing them via WingetUI from winget. As a fallback I've been using scoop apps which worked so far, tho they have separate config folders which isn't too ideal for my usecase.

I've experimented a bit and figured out that launching CMD/PS as admin does make the commands be recognized, as example "gallery-dl", but launching CMD/PS as a normal user doesn't. WingetUI isn't running as administrator either, so theoretically it should recognize winget installed apps as a normal user? Is maybe some path set up wrong during installation or due to the winget which ships with WingetUI?

Thanks already for your help!

**Comment by EpicLPer** [comment]
Funny enough: The path to "gallery-dl" was in the "User variables" section, but it just wasn't found by user launched Powershell or CMD. Once I put the same path into the system path section it also worked for my own account.

The weird thing I can't explain myself tho: If it was just in the user path how was the admin account able to find it? Don't think this is specifically a WingetUI problem but a general one of winget.

**Reply by marticliment** [reply]
If you manage to get thos working with winget-cli but not with WingetUI, please let me know and I'll investigate that

**Reply by EpicLPer** [reply]
For some reason on a freshly set up Win 11 VM I can't use winget directly, it simply shows the loading animation and then jumps right back to a prompt. I'll test this on another machine now tho.

EDIT: On my test-machine it works regardless if I install gallery-dl via PowerShell or WingetUI under user context... I'll see if I can test this further.
